pkg/imagefilter/formatter.go

```diff
@@ -99,18 +99,12 @@ type textShortResultsFormatter struct{}
 func (*textShortResultsFormatter) Output(w io.Writer, all []Result) error {
 	var errs []error
 
-	// deliberately break the yaml until the feature is stable, there
-	// are open questions, e.g. how this relates to:
-	// https://github.com/osbuild/osbuild-composer/pull/4336
-	// which adds a similar but slightly different API
-	fmt.Fprint(w, "@WARNING - the output format is not stable yet and may change\n")
```

Comment:
I would like to keep this for now, I think we need to consider what API we provide and grantee

Reply:
so I'll think about another code-comment as it's aligned with the 4336-PR…

Reply:
…and it doesn't state it's yaml, just happens to be compatible for now ;)

Reply:
It doesn't say a warning on the cloud-api though (and it's essentially the same output)
Should I transition to yaml library and remove the warning or make it really short and not making it yaml at all?
@supakeen / @mvo5

Reply:
So, with #1205 landing there's no reason anymore not to pull in a YAML dependency as it's already here.

My preference is that if the output format is outputting YAML that the format should be called `yaml` as well, same goes for the JSON format we have.

That means we can either, remove the YAML-ness in this PR and name it `short` or we can make it go full-YAML and name it `yaml`. No real preference on either.

Reply:
just pushes me towards making both :-D 
I'm fine with that ;)

Reply:
I am sorry that I missed this earlier but I think APis should be simple and orthogonal, I don't see the use-case for adding yaml in addition to json (happy to reconsider if there is one). 

As I said in https://github.com/osbuild/images/pull/1268#pullrequestreview-2645542052 I think "short" is very interesting (and making it yaml is nice as it is still very readable) because it can be a nice entry point (the current default is not very user friendly with  the wall of text that is the current default of list-images). But I think "short" should optimize for "niceness" (and IMHO we could sacrifice yaml for that even).

Reply:
There isn't a reason to have both. What I said in the previous comment:

- if the output is supposed to be YAML, use a YAML library
- if the output is not supposed to be YAML then let's make sure it can't be misconstrued as such

Not to introduce both.

Previously we were pretending that the `short` output format was YAML hence the warning up top to make sure YAML parsers don't understand it. That's also why I made my previous remarks about using a YAML library.